video_worker/__init__.py

```diff
@@ -97,25 +84,24 @@ def test(self):
         os.chdir(test_dir)
         test_bool = nose.run()
 
-        '''Return to previous state'''
+        # Return to previous state
         os.chdir(current_dir)
         return test_bool
 
     def run(self):
         WS = WorkerSetup(
             instance_yaml=self.instance_yaml
         )
-        if self.setup is True:
+        if self.setup:
             WS.setup = True
 
         WS.run()
         self.settings = WS.settings_dict
 
         if self.encode_profile is None:
-            ErrorObject().print_error(
```

Comment:
@muzaffaryousaf Are we keeping ErrorObject? we removed in previous 2 PRs though.